src/main/kotlin/com/ampnet/userservice/grpc/GrpcUserServer.kt

```diff
@@ -133,12 +140,41 @@ class GrpcUserServer(
         responseObserver.onCompleted()
     }
 
-    private fun getRole(role: SetRoleRequest.Role): UserRole =
+    override fun getAllActiveUsers(
+        request: PlatformManagerRequest,
+        responseObserver: StreamObserver<UsersWithExtendedInfoResponse>
+    ) {
+        logger.debug { "Received gRPC request getAllActiveUsers for coop: ${request.coop}" }
+        try {
+            validateUserIsPlatformManager(request)
```

Comment:
I wouldn't set restrictions in gRPC calls but rather verify user calls in Controller/Service calls from specific service.